app/components/status_tags/additional_document_component.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module StatusTags
+  class AdditionalDocumentComponent < StatusTags::BaseComponent
+    def initialize(planning_application:)
+      @planning_application = planning_application
+    end
+
+    private
+
+    attr_reader :planning_application
+
+    def status
+      if planning_application.additional_document_validation_requests.open_or_pending.any?
+        :invalid
+      elsif planning_application.documents_missing == false
```

Comment:
Future improvement to add 'not null' to all the boolean columns 😬

Reply:
Twenty Twenty Three problems